src/analyses/does_remove_const.cpp

```diff
@@ -12,9 +12,11 @@ Author: Diffblue Ltd.
 #include "does_remove_const.h"
 
 #include <goto-programs/goto_program.h>
-#include <util/type.h>
-#include <util/std_code.h>
+
 #include <util/base_type.h>
+#include <util/pointer_expr.h>
+#include <util/std_code.h>
+#include <util/type.h>
```

Comment:
Nit pick: I'm pretty sure `util/type.h` does not need to be included explicitly here.

Reply:
done